Programming Languages/Prolog/readme.md

```diff
@@ -22,7 +22,7 @@ Here are the collection of valuable resources to help you learn and master Prolo
 - [Youtube Videos](#youtube-videos)
 - [Books](#books)
 - [Community](#community)
-- [Additional Prolog Resources](#additional-prolog-resources)
+- [Additional Resources](#additional-resources)
```

Comment:
In additional Resources add the Books and Community since this is also done in other folders.